src/WebJobs.Extensions/Extensions/Timers/Listener/TimerListener.cs

```diff
@@ -342,7 +342,8 @@ internal async Task InvokeJobFunction(DateTime invocationTime, bool isPastDue =
                 if (ScheduleMonitor != null)
                 {
                     await ScheduleMonitor.UpdateStatusAsync(_timerLookupName, ScheduleStatus);
-                    _logger.LogDebug($"Function '{_functionLogName}' updated status: Last='{ScheduleStatus.Last.ToString("o")}', Next='{ScheduleStatus.Next.ToString("o")}', LastUpdated='{ScheduleStatus.LastUpdated.ToString("o")}'");
+                    _logger.LogDebug("Function '{functionName}' updated status: Last='{lastScheduled}\', Next=\'{nextScheduled}\', LastUpdated=\'{lastUpdated}\'", 
```

Comment:
nit, use formatter syntax:

```suggestion
                    _logger.LogDebug("Function '{functionName}' updated status: Last='{lastScheduled:o}\', Next=\'{nextScheduled:o}\', LastUpdated=\'{lastUpdated:o}\'", 
```

And then you can remove the `.ToString("o")` calls